src/views/Stake/Stake.jsx

```diff
@@ -0,0 +1,480 @@
+import { useCallback, useState } from "react";
```

Comment:
This file is duplicated by `Stake.tsx` & should be removed

Reply:
OK, I'll do that and resubmit.

Reply:
Done.